js/ui/components/clubhouse.js

```diff
@@ -752,7 +752,8 @@ var ClubhouseButtonManager = new Lang.Class({
             return;
 
         this._openButton.x = monitor.x + monitor.width - this._openButton.width / 2;
-        this._openButton.y = Math.floor(monitor.height / 2.0 - this._openButton.width / 2.0);
+        this._openButton.y =
+            Math.floor((monitor.height - Main.panel.height) / 2.0 - this._openButton.height / 2.0);
```

Comment:
Instead of looking at the panel height directly, can you use the layout manager API and use the primary monitor workarea? It should already have that computed including the panel (and whatever else may be affecting the workarea)